lib/hpke/include/hpke/certificate.h

```diff
@@ -53,4 +56,6 @@ struct Certificate
   const bytes raw;
 };
 
+bool operator==(const Certificate& lhs, const Certificate& rhs);
```

Comment:
const ?

Reply:
You mean `const bool`?  Seems unnecessary.

Reply:
Mainly `bool operatoor==(..) const;`